internal/dcache/cluster_manager/cluster_manager.go

```diff
@@ -88,4 +91,7 @@ type ClusterManager interface {
 
 	//Update RV state to offline and update MVs
 	reportRVFull(rvName string) error
+
+	//Notify consumer about cluster manager Event
+	notifyUpdates() <-chan dcache.ClusterManagerEvent
```

Comment:
getClusterMapUpdateNotificationChannel() is a more appropriate name, as the caller will call this to get the notification channel for getting clustermap update notifications and then read from the channel for any notifications.